redash/monitor.py

```diff
@@ -18,29 +23,20 @@ def get_object_counts():
 
 
 def get_queues():
-    queues = {}
-    for ds in models.DataSource.query:
-        for queue in (ds.queue_name, ds.scheduled_queue_name):
-            queues.setdefault(queue, set())
-            queues[queue].add(ds.name)
+    query = models.db.session.execute(
+        'select distinct queue_name from data_sources union all select distinct scheduled_queue_name from data_sources')
```

Comment:
In case you want to write this a Python 😁: 
```python
queue_names = db.session.query(DataSource.queue_name).distinct()
scheduled_queue_names = db.session.query(DataSource.scheduled_queue_name).distinct()
query = models.db.session.execute(union_all(queue_names, scheduled_queue_name))
```

Reply:
Thanks!

Reply:
212dcbb9b716b5efa3fcfa2bb626bbb70d974ac5

Although in retrospect I'm not sure what we gained by converting it to Python.